docs/first-order/modality_alignment.ipynb

```diff
@@ -0,0 +1,600 @@
+{
```

Comment:
<p>we "use".</p><p> </p><p>Choosing an embargoed data for this is a bit detrimental to the notebook. This will likely be unembargoed late. Can we use something else?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>